src/util/ResourceUtil.ts

```diff
@@ -33,3 +38,14 @@ export function generateContainmentQuads(containerURI: NamedNode, childURIs: str
   return new RepresentationMetadata(containerURI,
     { [LDP.contains]: childURIs.map(DataFactory.namedNode) }).quads();
 }
+
+export async function cloneRepresentation(representation: Representation): Promise<BasicRepresentation> {
+  const data = await arrayifyStream(representation.data);
+  const result = new BasicRepresentation(
+    data,
+    new RepresentationMetadata(representation.metadata, representation.metadata.contentType),
```

Comment:
The second parameter is not necessary as it is also going to inherit the content type by copying the input metadata.